src/chocolatey.console/choco.exe.manifest

```diff
@@ -5,32 +5,31 @@
     <security>
       <requestedPrivileges xmlns="urn:schemas-microsoft-com:asm.v3">
         <!-- UAC Manifest Options
-            Specifying requestedExecutionLevel node (below) will disable file and 
+            Specifying requestedExecutionLevel node (below) will disable file and
              registry virtualization.
-           
+
             Set level to "asInvoker" when you want choco to run under the context
-             of the user invoking. This is the same as Chocolatey behavior prior 
-             to 0.10.4 and after 0.10.7.
-             
-            Set level to "requireAdministrator" to only allow administrators to 
+             of the user invoking.
+
+            Set level to "requireAdministrator" to only allow administrators to
              run Chocolatey.
-            
-            Set level to "highestAvailable" to allow non-admins to run in 
+
+            Set level to "highestAvailable" to allow non-admins to run in
              non-administrative context, but require administrators to always run
              with administrative privileges. This is the recommended and default
-             behavior of Chocolatey v0.10.4-0.10.6.1.
-            
-            NOTE: Currently you will need to make this change every time 
+             behavior of Chocolatey.

```

Comment:
This says it's the default behaviour... but the setting is `asInvoker` ?

Reply:
You're right, this shouldn't be listed as the default behaviour, as it hasn't been since pre-1.0.

Reply:
Removed the mention of it being the default behaviour.